dietpi/dietpi-software

```diff
@@ -4199,6 +4205,111 @@ _EOF_
 			G_EXEC sysctl -w net.core.rmem_max=2500000
 		fi
 
+		if To_Install 206 # microblog.pub
+		then
+			## 1. Create a python environment py3.10 via pyenc
+			##	  see: https://python-poetry.org/docs/managing-environments/
+			local micro_pyenv_dir="/home/dietpi/.pyenv_microblog"
+			local micro_pyenv_activate_file="$micro_pyenv_dir/pyenv-activate.sh"
+			local micro_python_version='3.10.7'
+			local micro_dir="/home/dietpi/microblog"
+			local micro_virtualenv="/home/dietpi/microblog/.virtual_path"
+			local micro_systemd="/etc/systemd/system/microblog-pub.service"
+			local micro_functions="/etc/bashrc.d/microblog_pub.sh"
+
+			if ! [[ -d "$micro_dir" ]]; then
+				G_EXEC_OUTPUT=1 G_EXEC git clone https://git.sr.ht/~tsileo/microblog.pub "$micro_dir"
```

Comment:
Does it use Git internally as well? Else I suggest downloading the Git archive instead to skip the need to installing Git (`aSOFTWARE_DEPS[$software_id]='17'`) and the download and processing overhead.